azure-ipam/azilium.conflist

```diff
@@ -0,0 +1,14 @@
+{
+    "cniVersion": "0.3.1",
+    "name": "cilium",
```

Comment:
we need to change the name here, otherwise Cilium won't invoke cmdDel from azure-ipam. Pinging @tamilmani1989 and @wedaly

Reply:
apparently, the name conflicts with another existing chaining plugin's name, which makes the cmdDel flow returns early

Reply:
https://github.com/cilium/cilium/blob/8bcc5fe2c594b5fb4b9d2966906c490dea215d83/plugins/cilium-cni/cilium-cni.go#L664

Reply:
I think "cilium" will work, actually -- it's "azure" that was causing problems. The `chainingapi.DefaultConfigName` in the CNI code is "cilium", so setting name="cilium" tells the CNI to skip chaining (which os what we want).